assets/js/modules/analytics/util/is-zero-report.test.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Tests for report utilities.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies.
+ */
+import { isZeroReport } from './is-zero-report';
+
+describe( 'isZeroReport', () => {
+	it( 'returns undefined when undefined is passed', () => {
+		expect( isZeroReport( undefined ) ).toBe( undefined );
+	} );
+
+	it.each( [
+		[ 'NULL', null ],
+		[ 'FALSE', false ],
+		[ 'a number', 1 ],
+		[ 'a string', 'test' ],
+		[ 'an empty object', {} ],
+		[ 'an object without rows or totals', [ { data: {} } ] ],
+		[ 'an object with empty rows', [ { data: { rows: [] } } ] ],
+		[ 'an object with empty totals', [ { data: { totals: [] } } ] ],
+	] )( 'returns true when %s is passed', ( _, report ) => {
+		expect( isZeroReport( report ) ).toBe( true );
+	} );
+
+	it( 'returns true when the sum of totals values is 0', () => {
+		const report = [
+			{
+				data: {
+					rows: [
+						{}, {}, {},
+					],
+					totals: [
+						{
+							values: [
+								'0',
+								'0',
+							],
+						},
+					],
```

Comment:
Not to say that we should mirror our real/expected usage in tests (we shouldn't) but we should cover the cases that we use which aren't all covered here yet.
* Multiple items in `totals` (most of the time we just have 1, but some have 2, depending on the number of date ranges requested – 2 will be more common in the future)  
This also relates to my previous comment about whether or not we should consider values from other totals or only the first one (pending input from @felixarntz) but is worth including here either way.
* A single value in the list of `values` (depends on the number of metrics requested, but we usually only request one per report it seems)

Reply:
I've added in tests for multiple totals objects and single, multiple and mixed 0 and non-0 values.